models/amity.py

```diff
@@ -0,0 +1,428 @@
+import random
+from rooms import Office, Living
+from people import Staff, Fellow, StaffFromDatabase, FellowFromDatabase
+from termcolor import colored
+
+spacer = " "
+border = colored("*" * 5, 'cyan').center(80)
+
+
+class Amity(object):
+    def __init__(self):
+        self.rooms = []
+        self.vacant_rooms = []
+        self.offices = []
+        self.vacant_offices = []
+        self.livingspaces = []
+        self.vacant_livingspaces = []
+        self.people = []
+        self.allocated_people = []
+        self.unallocated_people = []
+        self.fellows = []
+        self.allocated_fellows = []
+        self.staff = []
+        self.allocated_staff = []
+
+    def create_room(self, args):
```

Comment:
**Code Quality: Critical**

This method definition seems awful similar to the next method. Keep your code DRY as much as possible.